demos/noise_suppression_demo/python/noise_suppression.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+"""
+ Copyright (c) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import logging as log
+import sys
+import time
+from argparse import ArgumentParser, SUPPRESS
+from pathlib import Path
+
+import numpy as np
+import soundfile
+
+from openvino.inference_engine import IECore
+
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
+    args.add_argument("-m", "--model", help="Required. Path to an .xml file with a trained model",
+                      required=True, type=Path)
+    args.add_argument("-i", "--input", help="Required. Path to a 16kHz sound file with speech+noise",
+                      required=True, type=str)
+    args.add_argument("-o", "--output", help="Required. Path to output sound file with speech",
```

Comment:
although not insisting, but it seems `-o` option might be made an optional, with some default file name, something like `noise_suppression_demo_out.wav`, for example.

Reply:
ok